.gitignore

```diff
@@ -26,7 +28,7 @@
 /plugins/wp-migrate-db-pro
 /plugins/wp-migrate-db-pro-media-files
 /themes/Beans
-/themes/developers/node_modules/
+/themes/developers/dev/node_modules/
```

Comment:
@rgadon107 This is unnecessary. The theme has a `.gitignore` that already excludes the node modules. So you can just delete this line. :)

Reply:
@hellofromtonya I removed the reference & updated the .gitignore for the child theme.